src/amo/components/Header/index.js

```diff
@@ -68,12 +69,18 @@ export class HeaderBase extends React.Component {
             ? <h1 className="Header-title-wrapper">{headerLink}</h1>
             : headerLink}
         </div>
+        <SectionLinks location={location} />
         <SearchFormComponent pathname="/search/" query={query} />
       </header>
     );
   }
 }
 
+export function mapStateToProps(state) {
+  return { isHomePage: state.currentView.isHomePage };
```

Comment:
It looks like [`amo/components/App/index`](https://github.com/mozilla/addons-frontend/blob/8a0c7baa68f8b0824fe93b6e91914159521c674d/src/amo/components/App/index.js#L196) also needs to be updated to get `isHomePage` from state.